programs/server/users.xml

```diff
@@ -27,6 +27,38 @@
         </readonly>
     </profiles>
 
+    <!-- External user directories (LDAP). -->
+    <ldap_servers>
+        <!-- List LDAP servers with their connection parameters here to later use them as authenticators for dedicated users,
+             who have 'ldap' authentication mechanism specified instead of 'password'.
+             Parameters:
+                host - LDAP server hostname or IP, this parameter is mandatory and cannot be empty.
+                port - LDAP server port, default is 636 if enable_tls is set to true, 389 otherwise.
+                auth_dn_prefix, auth_dn_suffix - prefix and suffix used to construct the DN to bind to.
+                        Effectively, the resulting DN will be constructed as auth_dn_prefix + escape(user_name) + auth_dn_suffix string.
+                        Note, that this implies that auth_dn_suffix should usually have comma ',' as its first non-space character.
+                enable_tls - flag to trigger use of secure connection to the LDAP server.
+                        Specify 'no' for plain text (ldap://) protocol (not recommended).
+                        Specify 'yes' for LDAP over SSL/TLS (ldaps://) protocol (recommended, the default).
+                        Specify 'starttls' for legacy StartTLS protocol (plain text (ldap://) protocol, upgraded to TLS).
+                tls_cert_verify - TLS peer certificate verification behavior.
+                        Accepted values are: never, allow, try, demand.
+                ca_cert_dir - path to CA certificates directory.
+                ca_cert_file - path to CA certificate file.
```

Comment:
Mention defaults (if any)

Reply:
No defaults, if not specified - not set via OpenLDAP. OpenLDAP uses OpenSSL, so some default behavior is there, but that story is too long to put it here.